deployment/pom.xml

```diff
@@ -3,17 +3,17 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.tietoevry.quarkus</groupId>
+        <groupId>io.quarkiverse.resteasy-problem</groupId>
         <artifactId>quarkus-resteasy-problem-parent</artifactId>
-        <version>3.9.1-SNAPSHOT</version>
+        <version>999-SNAPSHOT</version>
     </parent>
 
     <artifactId>quarkus-resteasy-problem-deployment</artifactId>
     <name>Quarkus - RESTeasy - Problem - Deployment</name>
 
     <dependencies>
         <dependency>
-            <groupId>com.tietoevry.quarkus</groupId>
+            <groupId>io.quarkiverse.resteasy-problem</groupId>
```

Comment:
Just asking: is `io.quarkiverse.resteasy` restricted somehow?
I'd prefer to have `groupId` like ☝️ (avoiding word redudancy), or maybe at least `io.quarkiverse.resteasy.problem` 🤔

Reply:
I had similar thoughts on this, but there seems to be some consensus on this in the docs: https://hub.quarkiverse.io/checklistfornewprojects/#start-coding
```
mvn io.quarkus.platform:quarkus-maven-plugin:create-extension \
 -DgroupId=io.quarkiverse.[myext] -DextensionId=[my-ext] -U
```
If we assume our extension id is `resteasy-problem` then it gives us:
```
<groupId>io.quarkiverse.resteasy-problem</groupId>
<artifactId>quarkus-resteasy-problem</artifactId>
```

this name is used in few other places like ecosystem ci repo, tooling etc